src/manifolds/VectorBundle.jl

```diff
@@ -272,8 +272,10 @@ function exp!(B::VectorBundle, q, p, X)
     xp, Xp = submanifold_components(B.manifold, p)
     xq, Xq = submanifold_components(B.manifold, q)
     VXM, VXF = submanifold_components(B.manifold, X)
-    exp!(B.manifold, xq, xp, VXM)
-    vector_transport_to!(B.manifold, Xq, xp, Xp + VXF, xq, B.vector_transport.method_point)
+    # this temporary avoids overwriting `p` when `q` and `p` occupy the same memory
+    xqt = exp(B.manifold, xp, VXM)
```

Comment:
We could also use `vector_transport_direction!` here, instead?
It does exactly what you do here, per default.

Reply:
Yes, sure -- I didn't think about that.

Reply:
Well, that‘s what reviews are for – and you fixed all errors, thanks for that! (I do not know why I saw errors with Product, maybe I was wrong? Ah, maybe I just saw a ProductRepr...)

Reply:
Done :slightly_smiling_face: . The product manifold looks fine, you probably just saw `ProductRepr` in `VectorBundle` tests.

Reply:
Perfect, then this one has just to wait for the decorator update (that is the larger one, I want to finish that first) – ah and I can remodel the deepcopies to copy tomorrow.